docs/api/system.lua

```diff
@@ -220,7 +272,9 @@ function system.sleep(seconds) end
 ---Similar to os.execute() but does not return the exit status of the
 ---executed command and executes the process in a non blocking way by
 ---forking it to the background.
+---Note: Do not use this function, use the Process API instead.
 ---
+---@deprecated
```

Comment:
Why mark this as deprecated? sometimes the complexity of using the process api isn't needed, eg: when calling open, xdg-open, start, etc...